clients/src/main/java/org/apache/kafka/clients/consumer/internals/NetworkClientDelegate.java

```diff
@@ -230,6 +234,10 @@ private ClientRequest makeClientRequest(
             unsent.handler
         );
     }
+    
+    public CompletableFuture<RuntimeException> metadataError() {
+        return metadataException;
+    }
```

Comment:
Any reason not to match the names?

```suggestion
    public CompletableFuture<RuntimeException> metadataException() {
        return metadataException;
    }
```

Reply:
I miss it, I will update it